Automattic/wp-calypso
# User licensing: Add post-checkout UI for activation "completed"

#### Changes proposed in this Pull Request

This is the post-checkout thank-you UI for when the user-licensing auto-activation flow is "completed".

Related to 1201096622142517-as-1201096624355406

**Additional notes:**

1. The figma designs are unclear about exactly how we are handling the 'unsuccessful' activation case.  We may need to further address the 'unsuccessful activation` case in a followup PR.
1. The figma designs are also unclear where exactly the "Go to Dashboard" button should lead for _each product_. When reviewing this PR, review all the `buttonUrl` locations in `use-get-jetpack-activation-confirmation-info.ts` and comment if you feel the button locations should be changed.

#### Testing instructions



- Create a JN site and connect Jetpack.
- Download this PR
- Start Calypso with `yarn start`.
- Visit https://cloud.jetpack.com/pricing.
- Select a paid product.
- Once you're on the checkout page, replace `wordpress.com/` with `calypso.localhost:3000/`.
- Submit the checkout form.
- Once you're on the page where a dropdown with your sites is displayed, select the JN site you connected in step 1.
- Verify the activation was successful and you can see the "Your [product name] is active!" UI, and it is consistent with the design in the screenshot below and in figma: ceKxiVna5wfpmwFBsojrAx-fi-2219%3A24970
- Verify the "Go to Dashboard" button leads to a reasonable location, considering the product just purchased. (See "Additional notes", item 2, above).
- To view the UI design for each product, in the url, replace the product slug with other jetpack product slugs (ie.- `jetpack_complete`, `jetpack_security_daily`, `jetpack_scan`, `jetpack_search`, `jetpack_anit_spam`, `jetpack_backup_daily`, etc) and reload the page.
- For each product, verify the UI text(copy) and the image (footerImage) matches the confirmation screens shown here: ceKxiVna5wfpmwFBsojrAx-fi-2531%3A27516

Test incompatible products error:
- Make another purchase **for the same product** you purchased originally in the steps above.
- Once you're on the page where a dropdown with your sites is displayed, **select the same JN site** you connected in step 1.
- Click "Continue" and verify that you see an error message that says, "I'm sorry, you cannot activate [product_name] on [selectedSite] because that site already has a subscription to [theIncompatibleProductName]." (see screenshot)

![Markup 2021-11-09 at 15 35 39](https://user-images.githubusercontent.com/11078128/141002797-291371e7-1e5a-4ed7-951f-f2b4a5636b59.png)

.
### Screenshots

![Screenshot on 2021-11-08 at 08-50-48](https://user-images.githubusercontent.com/11078128/140753760-3c16e817-db23-4926-bb10-9b1afe6d9024.png)
